inference-engine/src/inference_engine/src/ie_network_reader.cpp

```diff
@@ -333,16 +333,19 @@ CNNNetwork convert_to_cnnnetwork(std::shared_ptr<ngraph::Function>& function,
                     networkLayout << old_api_transpose_args[i];
                 }
 
+                PreProcessSteps steps;
+                // TODO: remove explicit type
```

Comment:
Looks like it is desired for you to have implicit conversion of element type and layout here, right?

Reply:
yes, like for `convert_layout` we have option with no args, the same we want here